src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -1704,4 +1704,14 @@ public String getTextFromComment(Text commentText) {
         }
         return SanitizationHelper.sanitizeForCsv(comment.toString());
     }
+
+    public boolean containsResponsesToInstructorOrGeneral() {
+        for (FeedbackResponseAttributes response : responses) {
+            String recipientIdentifier = response.recipient;
+            if (getSectionFromRoster(recipientIdentifier) == Const.NO_SPECIFIC_RECIPIENT) {
```

Comment:
Can u check the usage for `Const.NO_SPECIFIC_RECIPIENT`. I believe although the code is there, it not has been used (Tried to change `Const.NO_SPECIFIC_RECIPIENT ` to `Const.NO_SPECIFIC_SECTION` in `getSectionFromRoster ` on my CI https://travis-ci.org/xpdavid/teammates/builds/357798558?utm_source=github_status&utm_medium=notification). To me, the comparison is a bit confusing. You are getting the section from the roster but u are comparing it with `NO_SPECIFIC_RECIPIENT`